graphics/blit-atari.cpp

```diff
@@ -40,24 +41,31 @@ void Surface::create(int16 width, int16 height, const PixelFormat &f) {
 	w = width;
 	h = height;
 	format = f;
-	pitch = w * format.bytesPerPixel;
+	pitch = (w * format.bytesPerPixel + 15) & 0xfff0;
```

Comment:
Using ~15 ensures that if pitch is changed, the mask won't have to be changed.

Reply:
Sorry, I don't follow, where would you put that `~15` ? Or instead of what?

Reply:
Instead of masking with 0xfff0, mask with ~15 (or ~0xf).
This makes sure the bits are extended all along.

Reply:
Right, got it.